grpc/server/src/main/java/io/helidon/grpc/server/GrpcServerImpl.java

```diff
@@ -111,9 +129,25 @@
         String sName = config.name();
         int port = config.port();
         boolean tls = false;
+        SslConfiguration sslConfig = config.sslConfig();
+        SslContext sslContext = null;
 
         try {
-            NettyServerBuilder builder = NettyServerBuilder.forPort(port);
+            if (sslConfig != null){
```

Comment:
There should be a space before opening braces (and after closing brace before else statement) in this block